reactor-core/src/main/java/reactor/core/publisher/Flux.java

```diff
@@ -5522,8 +5522,10 @@ public int getPrefetch() {
 	}
 
 	/**
-	 * Map values from two Publishers into time windows and emit combination of values
-	 * in case their windows overlap. The emitted elements are obtained by passing the
+	 * Combine values from two Publishers in case their windows overlap. Each incoming
+	 * value triggers a creation of a new Publisher {@link Function}. If the
```

Comment:
a tiny bit misleading: change `{@link Function}` to `via the given {@link Function}`